standard/sections/clause_7_normative_text.adoc

```diff
@@ -329,13 +341,11 @@ include::../requirements/core/REQ_record_update_date.adoc[]
 
 ==== WMO Data Policy
 
-Based on the WMO Unified Data Policy Resolution (Res.1) footnote:[https://library.wmo.int/doc_num.php?explnum_id=11113#page=9], exchanged data
-are expressed as `+core+` or `+recommended+`.
+Based on the WMO Unified Data Policy for the International Exchange of Earth System Data (Resolution 1 (Cg-Ext(2021) footnote:[https://library.wmo.int/doc_num.php?explnum_id=11113#page=9], exchanged data are classified as `+core+` or `+recommended+`.
 
 include::../requirements/core/REQ_data_policy.adoc[]
 
-If the resource is classified as `+recommended+`, it needs to be addressed if any additional conditions apply.  Conditions on use
-are essential by the WMO Unified Data Policy, but for transparency and clarification any other conditions should also be addressed.
+If the resource is classified as `+recommended+`, it needs to be indicated if any additional conditions apply.  Conditions on use are essential for the WMO Unified Data Policy, but for transparency and clarification any other conditions should also be indicated.
```

Comment:
Just a cosmetic thing - there seems to be one extra space in front of "Conditions".